### PR TITLE
[Network Config Mgr] Integrate NMM and NCM.

### PR DESCRIPTION
**Pull request by pkommoju**
NMM and NCM are integrated but NMM still has no persistence.
Integration test script is updated to verify create, delete.
File upload is working.
There is a strange behaviour in addItemBulkTransaction(), come of the
log lines are not executed. These lines and other debug logs about
this problem are commented out in the PR for the record. This problem may
need to be fully understood but the commented code will be removed in the
next PR.

### CODE REVIEW

**Comment by xieus** [comment]
@pkommoju This PR appears to include some content from a merged PR #582. Do you create a new branch for this PR?

**Comment by pkommoju** [comment]
I did not create a new branch. I didn't see the latest change in the old PR getting, so I ended up creating a new PR.

**Comment by xieus** [comment]
> I did not create a new branch. I didn't see the latest change in the old PR getting, so I ended up creating a new PR.

Okay, git merge master would help then.

**Comment by pkommoju** [comment]
If something goes wrong, I can fix it in a new branch since my changes are not expected to overlap anyothers'.

File: scripts/json/nodes-upload.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+   "host_infos": [
+      {
+         "local_ip": "30.213.43.10",
+         "node_id": "e109465c-5ed6-400e-806d-fbefd12f94a9",
+         "mac_address": "D1:A0:C9:34:C8:29",
+         "node_name": "upl009030",
+         "server_port": 50001,
+         "veth": "eth0"
+      },
```
Comment (xieus) [inline scripts/json/nodes-upload.json:10]: Somehow pkommoju:nmm_ncm_api_07 branch is not in sync with the master, for example, node-upload.json file is an existing file in alcor:master branch. https://github.com/futurewei-cloud/alcor/blob/master/scripts/json/nodes-upload.json

Please fix your branch before calling for a review. The same issue repeatedly show up for 3 or 4 PRs. @pkommoju

**Comment by pkommoju** [comment]
Branch will be deleted next week.

**Pull request closed without merging**